docusaurus/docs/dev-docs/plugins/server-api.md

```diff
@@ -147,45 +147,6 @@ Once defined, the configuration can be accessed:
 Run `yarn strapi console` or `npm run strapi console` to access the strapi object in a live console.
 :::
 
-## Cron
```

Comment:
Not sure why it's been included in this PR 🤷 
It's in theory already included in `next` 🤔https://github.com/strapi/documentation/commit/8bd9b42931b4efeda66102d404d0c5081ecf1259